Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -94,11 +94,27 @@ def putMaskOnVTKGrid(data, grid, actorColor=None, cellData=True, deep=True):
             grid.GetPointData().AddArray(ghost)
     return mapper
 
+def handleProjectionEdgeCases(projName, data):
+    # For mercator projection, latitude values of -90 or 90
+    # transformation result in infinity values. We chose -85, 85
```

Comment:
No 85.0 is close enough for any practical purpose, I was just pointing out that it isn't a magic number chosen arbitrarily.  It is derived for a reason.